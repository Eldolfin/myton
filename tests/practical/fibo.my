def fibo(n):
  cur = 1
  next = 1
  while n > 0:
    next = cur + next
    cur = next - cur
    n = n - 1
  return cur
for i in [10, 20, 42]:
  print(fibo(i))
