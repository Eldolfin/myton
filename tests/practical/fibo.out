89
10946
433494437
